Gemfile

```diff
@@ -12,14 +12,16 @@ def location_for(place, version = nil)
   end
 end
 
-group :development, :unit_tests do
-  gem 'json',                      :require => false
-  gem 'metadata-json-lint',        :require => false
-  gem 'puppet_facts',              :require => false
-  gem 'puppet-blacksmith',         :require => false
-  gem 'puppetlabs_spec_helper',    :require => false
-  gem 'rspec-puppet', '>= 2.3.2',  :require => false
-  gem 'simplecov',                 :require => false
+group :development do
+  gem 'pry',  :require => false
+end
+group :unit_tests do
+  gem 'metadata-json-lint',                :require => false
+  gem 'puppet_facts',                      :require => false
+  gem 'puppet-blacksmith',                 :require => false
+  gem 'puppetlabs_spec_helper', '~> 1.1',  :require => false
+  gem 'rspec-puppet', '>= 2.3.2',          :require => false
```

Comment:
I'm a bit confused as to why `rspec-puppet`, the spec helper and `puppet_facts` aren't considered as part of the `development` group. Perhaps it should be the other way around and `pry` should be in a `debugging` group instead.